css-brochure-form/styles.css

```diff
@@ -38,7 +38,7 @@ select {
 
 label {
   padding: 10px 0 3px 10px;
-  font-family: 'Brandon Grotesque';
+  font-family: 'Brandon Grotesque', sans-serif;
   font-style: normal;
   font-weight: 500;
   font-size: 80%;
```

Comment:
These lines are only necessary in the `@font-face` rule, you can remove them here. I can't comment on the `src` line below, but it should be removed as well.